GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -1294,34 +1293,36 @@ private int writeRawDataGroupHeaders(Sheet sheet, QuestionGroupDto group, List<Q
         for (QuestionDto q : questions) {
             questionIdList.add(q.getKeyId().toString());
 
-            String questionId = q.getQuestionId();
-            // Can we tag the column with the variable name?
-            final boolean useQID = useQuestionId && questionId != null
-                    && !questionId.equals("");
+            String varName = q.getQuestionId();
+            // Can we tag the column(s) with the variable name?
+            final boolean doVarName = variableNamesInHeaders
```

Comment:
how about rename it to useVarName? or tagColumn, it is just it took me a few minutes to understand what doVarName actually did